tests/new_unit_tests/bootstrap.php

```diff
@@ -6,3 +6,10 @@
 require_once __DIR__ . '/../../vendor/autoload.php';
 
 \VCR\VCR::configure()->enableRequestMatchers(['method', 'url', 'body',]);
+
+
+// Override the default cache directory by setting an environment variable. This prevents our tests from overwriting
+// the user's real cache and session.
+// @TODO: Unit tests should not rely on side effects like these. When the Config object is properly injectable this
+// will not be necessary.
+putenv("TERMINUS_CACHE_DIR=~/.terminus/testcache");
```

Comment:
Perhaps this code could check for some other environment variable, e.g. TERMINUS_TEST_CACHE_DIR, and use that to overwrite TERMINUS_CACHE_DIR if the former is defined.  That way, folks who want to put this data somewhere else could do so, but things would still work right if run out-of-the-box.

Overall, seems like a big improvement; it would be good to get something like this in.